apps/web/src/services/asset-inventory/collector/shared/CollectorJobStatusIcon.vue

```diff
@@ -60,13 +60,15 @@ interface Props {
     contents?: string,
     status?: string,
     isArrow?: boolean,
+    tooltipPosition?: string,
 }
 
 const props = withDefaults(defineProps<Props>(), {
     to: '',
     contents: '',
     status: undefined,
     isArrow: false,
+    tooltipPosition: 'top-end',
```

Comment:
suggestion: This can be imported from here!

```typescript
import {POSITIONS} from "@spaceone/design-system/types/data-display/tooltips/type";
```